frontend/src/pages/articles/components/article-card/article-card.tsx

```diff
@@ -30,19 +48,29 @@ const ArticleCard: React.FC<Properties> = ({
   tags,
   reactions,
 }) => {
+  const user = useAppSelector(({ auth }) => auth.user) as UserAuthResponseDto;
   const { pathname } = useLocation();
+  const dispatch = useAppDispatch();
 
   const isMyArticles = matchPath(
     { path: `${AppRoute.ARTICLES}/${ArticleSubRoute.MY_ARTICLES}` },
     pathname,
   );
-  const { publishedAt, title, text, id, coverUrl } = article;
-  const { comments, views, likes, dislikes } = reactions;
+  const { publishedAt, title, text, id, userId, coverUrl } = article;
+  const MOCKED_READ_TIME = '7 min read';
+  const { likeCount, dislikeCount, isLike } = getReactionsInfo(
```

Comment:
Not sure if this `isLike` is not conflicting with param in handleReaction method